docs/tutorial.rst

```diff
@@ -248,6 +248,20 @@ response, and for each one, the response contains the following info:
   access_token when it expires. ``None`` unless explicitly requested.
   Details on refresh_token are in the next section
 
+.. note::
+
+    The keys into ``by_resource_server`` are the registered ``resource_server``
+    value for the service.
+    In general, the ``resource_server`` value will by the ID of the client used
+    in a service.
+
+    However, for Globus services like Globus Auth and Globus Transfer, the
```

Comment:
Feels a little odd to have Auth and Transfer be treated as exceptions to the general case?

Reply:
Maybe just move things around a bit? Something like:

For Globus hosted services like like Globus Auth and Globus Transfer, the ``resource_server`` value will be the hostname for the service.

For other services, including Globus Connect Server v5, the the ``resource_server`` value will be the ID of the service client.

Reply:
I'm just coming back to this and I agree with the suggestion. Let's put the services in as the "normal case" and everything else (the ID case) is "special". I'll also look at some of the other suggestions and probably fold it all together.

Reply:
The order and arrangement of the text I went with is a little different from Aaron's suggestion, since I think we need to go straight from "hostname of the service" to "and you can get that via `TransferClient.resource_server`".

Please take a look at the results and let me know what you think.